sketches/logo.js

```diff
@@ -99,7 +99,10 @@ const draw = p5 => {
 
 }
 
-const mousePressed = () => {
+const mousePressed = (p5) => {
+  if(isHovering(p5.mouseX, p5.mouseY)){
+    window.open("/","_self");
+  }
   toggle = !toggle;
```

Comment:
does this line (106) do anything now? should it move to the hover function?

Reply:
Yes we still need it. It still does the bring together effect if they click anywhere on the page